docs/services/vision/segmentation.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Segmentation (or 3D object segmentation)"
+linkTitle: "Segmentation"
+weight: 20
+type: "docs"
+description: "Select an algorith that creates point clouds of identified objects in a 3D image."
+tags: ["vision", "computer vision", "CV", "services", "segmentation"]
+# SMEs: Bijan, Khari
+---
+
+_3D Object Segmentation_ is the process of separating and returning a list of the identified "objects" from a 3D scene.
+The "objects" are a list of point clouds with associated metadata, like the label, the 3D bounding box, and center coordinates of the object.
+
+Any camera that can return 3D pointclouds can use 3D object segmentation.
+
+## Segmenter Types
+
+The types of segmenters supported are:
+
+- [**radius_clustering_segmenter**](#radius-clustering-segmenter): Radius clustering is a segmenter that identifies well separated objects above a flat plane.
+- [**detector_segmenter**](#detector-segmenter): Object segmenters are automatically created from detectors in the Vision Service.
+
+### Radius Clustering Segmenter
```

Comment:
```suggestion
### `radius_clustering_segmenter`
```

Like above, this is super optional. I think it looks nice in the sidebar/to navigate from types but it's also more formal to just spell it out

Reply:
If you do decide to make this change, would do the same for the other type